continuous_integration/install.sh

```diff
@@ -106,4 +106,6 @@ fi
 
 pip install nilearn
 
+pip install pandas
```

Comment:
For some reason, this does not works. Would you mind taking a look at 
https://travis-ci.org/nistats/nistats/builds/66196889 

AFAIK, since we do "conda update --yes conda" install.sh, pandas install is supposed to work... 
?? 

----- Mail original -----

> De: "Loïc Estève" <notifications@github.com>
> À: "nistats/nistats" <nistats@noreply.github.com>
> Cc: "bthirion" <bertrand.thirion@inria.fr>
> Envoyé: Jeudi 4 Juin 2015 15:03:37
> Objet: Re: [nistats] Dataframe (#3)

> In continuous_integration/install.sh :
> > @@ -106,4 +106,6 @@ fi
> >
> >  pip install nilearn
> >
> > +pip install pandas

> I don't think you need this, right? Pandas should be either installed via
> conda or apt-get in the neurodebian case.

> —
> Reply to this email directly or view it on GitHub .